extensions/positron-python/python_files/posit/positron/positron_jedilsp.py

```diff
@@ -792,20 +891,14 @@ def positron_help_topic_request(
     ),
 )
 def positron_code_action(
-    server: PositronJediLanguageServer,  # noqa: ARG001
-    params: CodeActionParams,  # noqa: ARG001
+    server: PositronJediLanguageServer,
+    params: CodeActionParams,
 ) -> Optional[List[CodeAction]]:
-    # Code Actions are currently causing the kernel process to hang in certain cases, for example,
-    # when the document contains `from fastai.vision.all import *`. Temporarily disable these
-    # until we figure out the underlying issue.
-
-    # try:
-    #     return code_action(server, params)
-    # except ValueError:
-    #     # Ignore LSP errors for actions with invalid line/column ranges.
-    #     logger.info("LSP codeAction error", exc_info=True)
-
-    return None
+    try:
```

Comment:
Re-enabled code actions too. I think we fixed the underlying issue a while ago. Although the current code actions aren't very useful tbh.